src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -196,13 +197,24 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
   private _computeColor(
     stateObj: HassEntity | LightEntity
   ): string | undefined {
+    if (stateObj.attributes.rgb_color && stateActive(stateObj)) {
+      return `rgb(${stateObj.attributes.rgb_color.join(",")})`;
+    }
     const iconColor = stateColorCss(stateObj);
     if (iconColor) {
       return `rgb(${iconColor})`;
     }
     return undefined;
   }
 
+  private _computeBrightness(stateObj: HassEntity | LightEntity): string {
+    if (stateObj.attributes.brightness && stateActive(stateObj)) {
+      const brightness = stateObj.attributes.brightness;
+      return `brightness(${(brightness + 245) / 5}%)`;
+    }
+    return "";
+  }
+
```

Comment:
I don't really like this code duplication, but was already present for the `_computeColor` as well, so did not change that in this PR.

Reply:
Yep we can change that in another PR 🙂